### ISSUE TITLE
[Markup and meta data] HTML markup isn't valid. (09.01.)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#41803]
####  VFS Team name
Profile

#### VFS product name
Personal Information

#### VFS feature name
Profile Personal Information Revision

#### Point of Contact/Reviewers
Sarah Koomson (@SarahKay8) - Accessibility

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
HTML markup isn't valid.

### Issue Details
"Gender Identity" Radio buttons needs fieldset and legend. Since the radio buttons are all part of a group, we can wrap them in a fieldset.  Although instructions are clear visually they aren't programmatically. Fieldset/legend is what makes that programmatic association. Within the fieldset, the legend tag would serve as a heading specifically for use in forms. Being a form element, JAWS or NVDA will read the contents of the legend tag before reading out the radio options. I would use "Select your gender identity" as the legend. 

### Link, screenshot or steps to recreate
Staging url: https://staging.va.gov/?next=%2Fprofile%2Fpersonal-information   
1. Navigate to the [Personal Information page](https://staging.va.gov/profile/personal-information) and log in with a test user.
2. Activate a screen reader. For testing I recommend NVDA on Windows (free to download) or VoiceOver on Mac (included with MacOS).
3. Press [Tab] until you reach the Gender Identity section's Edit button.
4. Press [Enter] to activate the button.
5. Focus is set to the first radio button in the list. The label for the radio button is announced, but the label on the question is not announced.	WCAG SC 2.5.3_A	
​

### VA.gov Experience Standard
[Category Number 09, Issue Number 01](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 1.3.1_A

---

### Platform Recommendation
Radio button needs fieldset and legend. Although instructions are clear visually they aren't programmatically. Fieldset/legend is what makes that programmatic association.   "For Developers -  <label for="some-id">` can only be associated with a single input's ID. Right now it's pointing to the `name` attribute used on all of the radio buttons (`root_genderIdentity`), but assistive technology like screen readers and voice command software will look for a matching `id` and not find an exact match.
​
Instead of using a `<label>`, use a `<fieldset>` around all of the radio button options and a `<legend>` for the question label. Something roughly like:
​
```
<fieldset>
<legend>Select your gender identity. (*Required)</legend>
​
<input type="radio" id="some-id_0" ... /> <label for="some-id_0" />Label text</label>
</fieldset> " 
```
cc @briandeconinck 

### VFS Team Tasks to Complete

- [ ] Comment on the ticket if there are questions or concerns
- [ ] Close the ticket when the issue has been resolved or validated by your Product Owner. If a team has additional questions or needs Platform help validating the issue, please comment in the ticket.

**Comment by adamwhitlock1** [comment]
This fix is now live on staging and ready to be validated @shiragoodman, @SKasimow and @Samara-Strauss 

![Screen Shot 2022-05-24 at 12.17.25 PM.png](https://images.zenhubusercontent.com/60ddebd504ce2452d0e0fa92/bffac79a-74b8-4009-904b-b644920ac73b)

If you right click on the 'Select your gender identity' and select 'inspect' you should be shown the markup in chrome devtools and the fieldset with legend will be present

**Comment by SarahKay8** [comment]
@adamwhitlock1 perfect! And super quick! Thanks Adam

**Comment by shiragoodman** [comment]
This may be a record for the fastest turnaround we've had on a launch-blocking issue. Great work Profile team! @adamwhitlock1 and/or @SKasimow - you can close this ticket.

**Comment by SKasimow** [comment]
@adamwhitlock1 here-here - nicely done Adam!!
